cfgov/jinja2/v1/_includes/on-demand/header.html

```diff
@@ -13,6 +13,14 @@
    remove this and empty after-header div #}
 <a href="#after-nav" id="skip-nav">Skip to main content</a>
 
+<div class="header-dom no-js">
 {% import 'organisms/header.html' as header with context %}
 {{ header.render( show_banner=false ) }}
+</div>
 <div id="after-nav"></div>
+
+<script>
+  // Limiting .no-js and .js to the header to avoid collisions elsewhere
+  var headerDom = document.querySelector( '.header-dom' );
```

Comment:
I was hesitant to touch anything related to the atomic organization for fear someone might grab it for use elsewhere. Since this should be temporary, I'd rather leave it as is so we remember to take it out later.